pyrft/other.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""
+ARI semi modified function files
+"""
+import numpy as np
+from scipy.ndimage import label
+from scipy.stats import norm
+
+from nilearn.input_data import NiftiMasker
+from nilearn.image import get_data, math_img
+
+def ari(pvalues, subset, alpha= 0.1 ):

```

Comment:
I haven't checked, but if this code is copy-pasted from Nilearn, you should rather depend on it (since you already have an dependence on Nilearn)

Reply:
I'm in the process of modifying the code to make it more suitable so it's not copy pasted.

Reply:
Well it is but is going to be different than the nilearn code is what I meant to say.

Reply:
Though this file may become deprecated if we're just going to use Simes to do that.